src/pages/Accounts/components/AccountItem.tsx

```diff
@@ -299,3 +334,18 @@ export const AccountItem = forwardRef(
 );
 
 AccountItem.displayName = 'AccountItem';
+
+const getCopyEventNameByNetworkType = (type: NetworkVMType) => {
+  switch (type) {
+    case NetworkVMType.BITCOIN:
+      return 'AccountSelectorBtcAddressCopied';
+    case NetworkVMType.EVM:
+      return 'AccountSelectorEthAddressCopied';
+    case NetworkVMType.AVM:
+      return 'AccountSelectorAVMAddressCopied';
+    case NetworkVMType.PVM:
+      return 'AccountSelectorPVMAddressCopied';
+    default:
+      return 'AccountSelectorAddressCopied';
+  }
```

Comment:
Are these older event names we need to follow? Can we do something like this instead?
```suggestion
return  `AccountSelector${type}AddressCopied`;
```

Reply:
We used two of them before: `Btc` and `Eth`

Reply:
Simplified a bit in 6b13e6aae + made the tooltip contents selectable.